src/CMakeLists.txt

```diff
@@ -68,6 +68,14 @@ target_include_directories(benchmark PUBLIC
     )
 target_link_libraries(benchmark_main benchmark::benchmark)
 
+# For configuring pkg-config benchmark.pc library name
+string(TOLOWER ${PROJECT_NAME} PROJECT_NAME_LOWER)
+if(CMAKE_BUILD_TYPE STREQUAL "Debug")
+  get_target_property(_BENCHMARK_LIB_POSTFIX ${PROJECT_NAME} DEBUG_POSTFIX)
```

Comment:
That is awfully indirect..
Can't we ask cmake what filename it will use, instead of guessing at it?

Reply:
It is but, but is this better?

get_target_property(OUT benchmark OUTPUT_NAME)

Reply:
I mean, this only accounts for the `POSTFIX` for debug build,
so if we add postfix for release build, this will need to be updated too.

What i'm specifically not happy about here is that we are **still** making a guess
about the library file name, instead of asking cmake about it.

Reply:
No I totally agree, I haven't found a simple way without generator expression?